js/views/chatVw.js

```diff
@@ -113,9 +130,13 @@ module.exports = baseVw.extend({
         filteredMd = this.filteredChatConvos.findWhere({ guid: md.get('guid') });
         filteredMd && filteredMd.set(md.attributes);
       }
-
-      md.hasChanged('unread') && this.setAggregateUnreadCount();          
-    });      
+      
+      if (md.hasChanged('unread') && md.get('unread') > 0) {
```

Comment:
hmm.. maybe we don't need to re-sort, since the request is for 'unread' message to float to the top -not- read messages to float down. Not sure..?

But, in either case, we do want the aggregate (task bar) counts to be properly updated.